### PR TITLE
Fix #730 (redundant INFO log)

### LINKED ISSUE #730: Redundant INFO log when starting server
**Is your feature request related to a problem? Please describe.**
I have the default log level set to INFO in my application. Unfortunately warp likes to log two log messages at startup that effectively mean the same thing. This isn't very important, but it gets a bit annoying.
```
[2020-10-23T04:04:16Z INFO warp::server]: Server::run; addr=127.0.0.1:7878
[2020-10-23T04:04:16Z INFO warp::server]: listening on http://127.0.0.1:7878
```

**Describe the solution you'd like**
Remove this first line:
```
[2020-10-23T04:04:16Z INFO warp::server]: Server::run; addr=127.0.0.1:7878
```
An alternative would be to change the log level of that message to DEBUG.

**Describe alternatives you've considered**
I could just roll with these log messages, or disable logging for INFO level messages from warp. (I don't know how to do the second one though.)

**Additional context**
Not much. N/A.

### PR DESCRIPTION
**Pull request by kotx**
Fixes #730.

### CODE REVIEW

File: src/server.rs [hunk 1]
```diff
@@ -131,7 +131,6 @@ where
     pub async fn run(self, addr: impl Into<SocketAddr>) {
         let (addr, fut) = self.bind_ephemeral(addr);
         let span = tracing::info_span!("Server::run", ?addr);
-        tracing::info!(parent: &span, "listening on http://{}", addr);
```
Comment (jxs) [inline src/server.rs:134]: can we maintain `listening on http://{}" instead of `Server::run` as it was the original log?

**Comment by jxs** [comment]
cc @hawkw can you take a look Eliza?

File: src/server.rs (hunk 1 above)
Comment (kotx) [reply src/server.rs:134]: To be honest, I have no idea what lines 133 and 135 do (not much experience with Rust or this codebase). Line 134 can be cleanly removed without any other code changes. Would changing L133 to `let span = tracing::info_span!("Listening on", ?addr);` while removing L134 work?

**Comment by thibault-martinez** [comment]
Would you consider putting this log in DEBUG ?
Most applications are actually already printing something in INFO with more information themselves, so it makes it a bit redundant.

**Comment by jxs** [comment]
Hi, sorry to get back to this so late. I just noticed running the `tracing` example on your branch warp doesn't log anything:
on `master` running the tracing example:
`Jun 11 13:29:35.199  INFO Server::run{addr=127.0.0.1:3040}: warp::server: listening on http://127.0.0.1:3040`
on `kotx:patch-1` nothing is logged. Can you provide the code that triggers #730 ?

**Comment by kotx** [comment]
> 
> 
> Hi, sorry to get back to this so late. I just noticed running the `tracing` example on your branch warp doesn't log anything:
> on `master` running the tracing example:
> `Jun 11 13:29:35.199 INFO Server::run{addr=127.0.0.1:3040}: warp::server: listening on http://127.0.0.1:3040`
> on `kotx:patch-1` nothing is logged. Can you provide the code that triggers #730 ?

My bad, seems this PR and the issue are outdated; the log is now one line instead of the two it was. Closing them now.

**Comment by jxs** [comment]
no worries :) thanks!

**Pull request closed without merging**